src/Options.php

```diff
@@ -940,4 +965,22 @@ private function validateMaxBreadcrumbsOptions(int $value): bool
     {
         return $value >= 0 && $value <= self::DEFAULT_MAX_BREADCRUMBS;
     }
+
+    /**
+     * Validates all the class serializers are callables indexed by class names.
```

Comment:
Change the description to `Validate that the value passed to the `class_serializers` option is valid.`